JavaScript/Popup.js

```diff
@@ -0,0 +1,189 @@
+  document.body.append(popup);
+  popup.classList.add('pop-up-class');
+  // add elements
+  const header = document.createElement('div');
+  header.classList.add('header-pp');
+  const headingtitle = document.createElement('h2');
+  headingtitle.classList.add('project-head');
+  const projectimg = document.createElement('img');
+  projectimg.classList.add('image-pp');
+  const positiondiv = document.createElement('div');
+  positiondiv.classList.add('position-div-pp');
+  const paragraphdescription = document.createElement('p');
+  paragraphdescription.classList.add('project-descrip');
+  const langbtndiv = document.createElement('div');
+  langbtndiv.classList.add('lang-btn-div');
+  const proLanguage = document.createElement('ul');
+  proLanguage.classList.add('lang-container-pp');
+  const projectbtncontainer = document.createElement('div');
+  const btnseelive = document.createElement('a');
+  const btnseesource = document.createElement('a');
+  const closeButton = document.createElement('button');
+  closeButton.classList.add('closeButton-class');
+  closeButton.id = 'close-button';
+  popup.append(
+    header,
+    headingtitle,
+    closeButton,
+    projectimg,
+    positiondiv,
+    paragraphdescription,
+    langbtndiv,
+    proLanguage,
+    projectbtncontainer,
+  );
+  header.append(headingtitle, closeButton);
+  positiondiv.append(paragraphdescription, langbtndiv, proLanguage, projectbtncontainer);
+  langbtndiv.append(proLanguage, projectbtncontainer);
+  projectbtncontainer.append(btnseelive, btnseesource);
+
+  const projectId = parseInt(event.target.id, 10);
+  headingtitle.textContent = ProjectInformation[projectId].name;
+  projectimg.src = ProjectInformation[projectId]['featured image'];
+  paragraphdescription.textContent = ProjectInformation[projectId].description;
+
+  for (let i = 0; i < ProjectInformation[projectId].technologies.length; i += 1) {
+    const projectLi = document.createElement('li');
+    projectLi.className = 'languages-pp';
+    projectLi.textContent = ProjectInformation[projectId].technologies[i];
+    proLanguage.appendChild(projectLi);
+  }
+  projectbtncontainer.classList.add('btn-container-class-pp');
+  btnseelive.href = ProjectInformation[projectId]['link to live version'];
+  btnseelive.classList.add('btn-seelive-pp');
+  btnseelive.target = '_blank';
+  btnseelive.textContent = 'See Live';
+  btnseesource.href = ProjectInformation[projectId]['link to source'];
+  btnseesource.classList.add('btn-seesource-pp');
+  btnseesource.target = '_blank';
+  btnseesource.textContent = 'See Source';
+  // close button function
+  function closeButtonFunction() {
+    popup.remove();
+  }
+  const clickCloseButton = document.getElementById('close-button');
+  clickCloseButton.addEventListener('click', closeButtonFunction);
+}
+// button
+const buttonsee = document.querySelectorAll('.btn-see-project');
+buttonsee.forEach((btn) => {
+  btn.addEventListener('click', makePopup);
+});
```

Comment:
- [ ] there's a bug where sometimes when I open a specific project I get the information about another project, kindly fix that issue so that when I click on a specific project I get only the information related to it  🙏 

![popup](https://user-images.githubusercontent.com/89282221/216047929-0033db2b-65f4-4c32-a1d6-4a8cfd600fdc.gif)